### ISSUE TITLE
[GTNH] Recipe conflict checker is broken. 

### ISSUE THREAD

**Issue by shadowpho** [Divran/greginator#4]
![image](https://user-images.githubusercontent.com/217332/209900473-9397c154-8da1-464f-b179-d11060417773.png)

After page refresh and uploading json:

![image](https://user-images.githubusercontent.com/217332/209900530-11d4b0ae-1031-4bed-987e-a52997715490.png)

**Comment by Divran** [comment]
Please try pressing Ctrl f5, which will clear your browser cache for this website specifically and do a full refresh. If the problem is still there then press f12 to open console and tell me if there are any errors. And if there are errors then please copy paste the json from the text box in the top right corner. It's very late for me right now so I won't be able to reply until tomorrow.

**Comment by shadowpho** [comment]
Thank you! No problem, it's free service that makes our game so much better =).

It's probably some upgrade that happened -- the json file breaks on new version, but works on old version.  Here is the json: https://pastebin.com/6Y9d6HQj

**Comment by Divran** [comment]
Fixed, somewhat. I had this feature previously but it wasn't working: basically the entire recipe is saved in the json, as it was at the time that you added it. then, if an update changes this recipe, it will still be able to load the OLD VERSION of that recipe, and it will tell you when it wasn't able to find a matching identical recipe. But like I said this wasn't working, which is part of why the whole list was just blank.

So that's fixed now, and it will tell you with big red colors when it finds a recipe that isn't correct. However, I also found a different problem. For some reason gtnh devs have made tiny changes to the oredict names of certain items, which will cause some recipes to not be found even though technically the recipes are still the same. To fix those you will probably just have to re-add the new version of the recipe and delete the broken one.

Also, a different side effect of the 2.2.8 update is, because recipes are placed into separate input buses based on the order they happen to appear in, and the order of recipes has changed in the GTNH update, the resulting suggested recipe separation will not be the same as in the old version. Not much I can do about that though, except to write a far more complex algorithm, and I'm probably not going to try to do that for a while.
